src/api/app/helpers/webui/user_helper.rb

```diff
@@ -1,4 +1,36 @@
 module Webui::UserHelper
+  def display_applied_filters(filters)
+    tag.span(class: 'd-block ml-4 mb-3') do
+      link_to(User.session!) do
+        tag.i(class: 'fas fa-trash-alt') do
```

Comment:
![Screenshot_2020-11-20 Profile of Admin](https://user-images.githubusercontent.com/2581944/99795876-d0688c00-2b2c-11eb-80e3-d9b404515e29.png)

Now there is some missing space between the icon and the link.

Reply:
fixed. I updated the screenshots to the last version.